### PR TITLE
Update shared

### PR DESCRIPTION
**Pull request by nicolai-nordic**
Update pc-nrfconnect-shared dependency to latest version (v4.13.0).

### CODE REVIEW

File: lib/components/HexOnlyEditableField.jsx [hunk 1]
```diff
@@ -230,26 +235,19 @@ class HexOnlyEditableField extends React.Component {
...
-            keyPressValidation,
-            completeValidation,
-            formatInput,
-            onBeforeBackspace,
-            onBeforeDelete,
-            onChange,
-            title,
-            ...props
-        } = this.props; // pass along all props except these
+        const { showText, value, title, ...props } = this.props; // pass along all props except these
```
Comment (bihanssen) [inline lib/components/HexOnlyEditableField.jsx:250]: Noticing that several props have disappeared here, assume it's deliberate?

**Review by bihanssen — COMMENTED** [review]
Good round of house cleaning. LGTM.

File: lib/components/HexOnlyEditableField.jsx (hunk 1 above)
Comment (nicolai-nordic) [reply lib/components/HexOnlyEditableField.jsx:250]: Yeah they were not used.

**Review by bihanssen — APPROVED** [review]

**Comment by datenreisender** [comment]
Thanks for the accomplishing to migrate the BLE app to the latest shared. There are some more aspects which are not strictly needed but it would help us if you could also take care of them:
- We are phasing out `lint-init` through all of our projects. To accomplish that in the BLE app, remove the scripts `lint-init` and `postinstall` from `package.json`, remove the entry `.eslintrc` from `.gitignore` and add these entries to `package.json`:
```json
  "eslintConfig": { "extends" : "./node_modules/pc-nrfconnect-shared/config/eslintrc.json" },
  "prettier": "./node_modules/pc-nrfconnect-shared/config/prettier.config.js",
```
- [Add a `.huskyrc.json`](https://github.com/NordicSemiconductor/pc-nrfconnect-shared/blob/master/Changelog.md#steps-to-upgrade-when-using-this-package-6) so tests and listing are done before pushing.
- In `azure-pipelines.yml` change the `npm i` to `npm ci` to make the build fail if one commits an outdated `package-lock.json`.

**Pull request merged**